spec/services/google/client_service_spec.rb

```diff
@@ -4,27 +4,27 @@
 
 RSpec.describe Google::ClientService, type: :service do
```

Comment:
We are missing it, all the testing code should be inside this block
```ruby
describe '#call' do
...
...
end
```

Reply:
Yup! :)